satpy/etc/composites/ami.yaml

```
@@ -22,19 +24,48 @@ composites:
     fractions: [0.85, 0.15]
 
   green_nocorr:
+    # Deprecated green composite using deprecated class 'GreenCorrector'. Use 'hybrid_green_nocorr' instead.
     compositor: !!python/name:satpy.composites.spectral.GreenCorrector
     prerequisites:
       - name: VI005
       - name: VI008
     standard_name: toa_reflectance
     fractions: [0.85, 0.15]
 
+  hybrid_green_raw:
+    compositor: !!python/name:satpy.composites.spectral.HybridGreen
+    prerequisites:
+      - name: VI005
+        modifiers: [ sunz_corrected ]
+      - name: VI008
+        modifiers: [ sunz_corrected ]
+    standard_name: toa_bidirectional_reflectance
+    fraction: 0.15
+
+  hybrid_green:
+    compositor: !!python/name:satpy.composites.spectral.HybridGreen
+    prerequisites:
+      - name: VI005
+        modifiers: [ sunz_corrected, rayleigh_corrected ]
+      - name: VI008
+        modifiers: [ sunz_corrected ]
```

Harmonize list syntax.
```suggestion
        modifiers: [sunz_corrected]
      - name: VI008
        modifiers: [sunz_corrected]
    standard_name: toa_bidirectional_reflectance
    fraction: 0.15

  hybrid_green:
    compositor: !!python/name:satpy.composites.spectral.HybridGreen
    prerequisites:
      - name: VI005
        modifiers: [sunz_corrected, rayleigh_corrected]
      - name: VI008
        modifiers: [sunz_corrected]
```